ang/civicase.ang.php

```diff
@@ -232,16 +232,17 @@
 $options['allowMultipleCaseClients'] = (bool) Civi::settings()->get('civicaseAllowMultipleClients');
 $options['allowCaseLocks'] = (bool) Civi::settings()->get('civicaseAllowCaseLocks');
 
-if ( ! function_exists('glob_recursive')) {
+if (!function_exists('glob_recursive')) {
   /**
    * Recursive Glob function
    * Source: http://php.net/manual/en/function.glob.php#106595
    * Does not support flag GLOB_BRACE
    */
   function glob_recursive($pattern, $flags = 0) {
     $files = glob($pattern, $flags);
-    foreach (glob(dirname($pattern).'/*', GLOB_ONLYDIR|GLOB_NOSORT) as $dir) {
-      $files = array_merge($files, glob_recursive($dir.'/'.basename($pattern), $flags));
+
+    foreach (glob(dirname($pattern) . '/*', GLOB_ONLYDIR|GLOB_NOSORT) as $dir) {
+      $files = array_merge($files, glob_recursive($dir. '/'. basename($pattern), $flags));
```

Comment:
I am afraid but we still need to add a space before and after `. '/'` 
It should be 
```php
($dir . '/' . basename($pattern), $flags));
```